### PR TITLE
Added test case for worker-ccai

### PR DESCRIPTION
**Pull request by bobvanmanen**
I had a difficult time matching the body of the request. I have come to the conclusion that maybe MockServer doesn't support `multipart/form-data`. The one example they have on form data (https://www.mock-server.com/mock_server/creating_expectations.html#button_match_request_by_form_submission_body) has a `application/x-www-form-urlencoded` content type.

I ended up using a substring matcher to verify that the JSON is part of the POST request. I'm not checking whether the file is in the POST request though.

### CODE REVIEW

File: projects/worker-ccai/test/asset-compute/worker-ccai/simple-test/params.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ }
```
Comment (tmathern) [inline projects/worker-ccai/test/asset-compute/worker-ccai/simple-test/params.json:1]: I think you don't need to keep an exmpty params file-object.

**Review by jdelbick — APPROVED** [review]
regarding matching the api request exacting using mock server:

we've gotten feedback that it is difficult to match the exact HTTP request for mock server (and sometimes the http request is hidden behind a node library like cloud-storage). we have a ticket to explore adding a general mocking framework where we can mock libraries instead of raw http requests.

**Comment by bobvanmanen** [comment]
> regarding matching the api request exacting using mock server:
> 
> we've gotten feedback that it is difficult to match the exact HTTP request for mock server (and sometimes the http request is hidden behind a node library like cloud-storage). we have a ticket to explore adding a general mocking framework where we can mock libraries instead of raw http requests.

MockServer provides a UI (https://www.mock-server.com/mock_server/mockserver_ui.html) that could help this, but for that to work MockServer has to run longer than the individual tests. I was looking for an ability to show you "unmatched" requests for a particular host. Nock can do this for NodeJS unit tests, and it makes iterating a lot easier. Maybe keeping MockServer running so the UI can be checked is enough though.

**Comment by alexkli** [comment]
@bobvanmanen 
> Maybe keeping MockServer running so the UI can be checked is enough though.

Ack, worth exploring. We deliberately picked an existing elaborate solution such as MockServer to benefit from its existing tooling.

cc @jdelbick

**Pull request merged**